packages/fluentui/docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -486,6 +487,7 @@ const Sidebar: React.FC<RouteComponentProps & SidebarProps> = props => {
         >
           Fluent <span style={gradientTextStyles}>UI</span>
         </Text>
+        <VersionDropdown width={props.width} />
```

Comment:
```suggestion
        {process.env.NODE_ENV === 'production' && <VersionDropdown width={props.width} />}
```

So we hide this during local builds?

Reply:
This is should already be covered in the component itself, although you reminded me that I have no `catch` for the fetch.... is that enough for you or should we be more explicit to not render this locally at all ? It might be necessary to locally host a manifest to display this dropdown if you're working on it (like I have)